src/Pages/PassengerInfo/Accordian.js

```diff
@@ -0,0 +1,240 @@
+import React, { useState, useRef } from 'react';
+import styled from 'styled-components';
+
+function Accordian({ title }) {
+  const content = useRef(null);
+  const [height, setHeight] = useState(0);
+  const toggleAccordion = () => {
+    setHeight(height === 0 ? content.current.scrollHeight + 30 : 0);
+  };
+  const checkInfo = () => {};
+
+  return (
+    <AccordianAll>
+      <AccordionHeading>
+        <Container>
+          <TitleStyle>{title}</TitleStyle>
+          <button onClick={toggleAccordion}>Click</button>
+        </Container>
+      </AccordionHeading>
+
+      <AccrodionContent height={height} ref={content}>
+        <Container1>
+          <NameText>
+            <FristName>
+              승객 성(영문)<Required>필수 입력</Required>
+              <InputAll type="text" />
+            </FristName>
+            <SecondName>
+              승객 이름(영문)<Required>필수 입력</Required>
+              <InputAll type="text" />
+            </SecondName>
+          </NameText>
+
+          <AddInfo>
+            <Gender>
+              성별<Required>필수 입력</Required>
+              <BtnRadio>
+                <Ckbox name="성별" type="radio" value="남자" />
```

Comment:
```suggestion
                <Checkbox name="성별" type="radio" value="남자" />
```
애매하게 줄이기 보다는 그냥 곧바로 써주는게 좋을 것 같아요